specs/v2-sketch.yaml

```diff
@@ -31,6 +31,51 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Project"
+  /project/{project_slug}/pipeline:
+    post:
+      summary: Trigger a new pipeline for a project
+      operationId: triggerNewPipeline
+      parameters:
+        - $ref: "#/components/parameters/ProjectSlug"
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: "#/components/schemas/TriggerPipelineRequest"
+          application/json:
+            schema:
+              $ref: "#/components/schemas/TriggerPipelineRequest"
+      responses:
+        '200':
+          description: The ID of the newly created pipeline
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - pipeline_id
```

Comment:
Per Slack discussion, let's go with `id` for all major entities rather than `foo_id` for an entity `foo`

Reply:
This is a slightly odd one, it's not a pipeline object but it is a map of information about a pipeline. The change still makes sense, so I'll make it.